### PR TITLE
Make the steps for deploying to Meta Quest 2 (and other Android headsets) clearer

### PR DESCRIPTION
**Pull request by dsnopek**
This PR is attempting to specifically address the points of confusion from https://github.com/godotengine/godot/issues/78427

Namely:

- Making it extra clear that you need to follow most of the normal Android setup steps before doing the things on the OpenXR-specific page.
- Changing one of the screenshots to have the "Use Gradle build" checkbox checked (as described in the text instructions, but confusingly shown unchecked in the original screenshot)
- Specifically linking to the "One-click deploy" documentation, and making it clear that developers may need to perform some extra steps from their headset vendor's documentation in order to make it work. And I added a note and link for the Meta Quest in particular, since it's the most popular headset.

I also fixed some terminology that I think is incorrect: "export template" -> "export preset"

### CODE REVIEW

**Comment by BastiaanOlij** [comment]
Looks good!

I would maybe add a screenshot of the dropdown and option to run, I could create one for you to use that shows both Quest and Pico in the menu to make clear why it is a dropdown.

**Comment by dsnopek** [comment]
Thanks :-)

> I could create one for you to use that shows both Quest and Pico in the menu to make clear why it is a dropdown

Yeah, that'd be awesome! Personally, I only have various flavors of Quest

**Comment by BastiaanOlij** [comment]
@dsnopek you'll need to crop it and probably convert it to webp, but hopefully this helps?

Godot Android VR dropdown

**Comment by dsnopek** [comment]
Thanks so much, @BastiaanOlij!

I cropped Bastiaan's screenshot and blurred the areas behind the dropdown (since they are pretty distracting), converted to webp and added to the new section about one-click deploy.

Hopefully, the blur is an OK way to do this? I could also just cut those parts out, but I worried that that could be confusing since there _is_ stuff there and might it make it hard for folks to find the right part of the screen.

**Comment by skyace65** [comment]
For a image like that where you want to direct people to a specific part you would add an outline instead of blurring the background. Like this:
[android_one_click_deploy.zip](https://github.com/godotengine/godot-docs/files/11900886/android_one_click_deploy.zip)

And if you don't mind please update your PR with that image.

**Comment by dsnopek** [comment]
Sure! I've updated the PR, replacing the image with yours. Thanks for making it :-)

**Review by BastiaanOlij — APPROVED** [review]
lgtm, nice work!

File: tutorials/xr/deploying_to_android.rst [hunk 1]
```diff
@@ -35,14 +44,15 @@ There is a subfolder called **plugins** containing the new files.
 
 You can find the main repository of the loader plugin `here <https://github.com/GodotVR/godot_openxr_loaders>`__.
 
-Creating the export templates
------------------------------
-You will need to setup a separate export template for each device as each device will need its own loader included.
+Creating the export profiles
```
[the remark this answers is not in the document]
Comment (BastiaanOlij) [reply tutorials/xr/deploying_to_android.rst:47]: Worth changing over, I'm probably to blame here with my broken English :P

[the remark this answers is not in the document]
Comment (dsnopek) [reply tutorials/xr/deploying_to_android.rst]: Heh, sorry, no it was my mistake :-)

I've fixed it in my latest push

**Review by skyace65 — APPROVED** [review]

**Comment by skyace65** [comment]
Thanks!

**Pull request merged**